libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -151,6 +151,7 @@ class AP_SerialManager {
         SerialProtocol_ADSB = 35,
         SerialProtocol_AHRS = 36,
         SerialProtocol_SmartAudio = 37,
+        SerialProtocol_Torqeedo = 38,
```

Comment:
```suggestion
        SerialProtocol_Torqeedo = 39,
```

For @amilcarlucas

Reply:
txs.  Torqeedo serial protocol changed to 39 (also updated the param description.